bescort.lic

```diff
@@ -289,12 +293,44 @@ class Bescort
     elsif args.hara_polo
       hara_polo(args.mode)
     elsif args.jolas
-      jolas(args.mode)       
+      jolas(args.mode)
+    elsif args.currach
+      currach(args.mode)
     end
   end
 
   private
 
+  def currach(mode)
+    if (mode.include?('aesry') && (Room.current.id == 5555 || Room.current.id == 5557 || Room.current.id == 5558) || (mode.include?('halasa') && Room.current.id == 15863))
+      echo "You're already there silly!"
+      exit    
+    elsif mode == 'aesry'
+      row_direction = 'pier'
+    elsif mode == 'halasa'
+      row_direction = 'rock'
+    else
+      echo("Unrecognized argument: #{mode}")
+      exit
+    end  
+
+    if (Room.current.id == 5555 || Room.current.id == 5557 || Room.current.id == 5558 || Room.current.id == 15863)
+      fput "go currach", "You climb aboard the", "You are aboard", "What are you referring"
+      fput "get oars"
+      fput "untie currach"
+    end
+
+    loop do
+      if bput("tie currach to #{row_direction}", "isn't close enough!", "You pull at the oars") == "You pull at the oars"
+        fput "stand"
```

Comment:
What do you think about using `fix_standing` here?

Reply:
Didn't even know about fix_standing. Updated on v2